File: c++/1012.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

int main(){

   double a, b, c, pi;

   cin >> a >> b >> c;
   pi = 3.14159;

   cout << fixed << setprecision(3);

   cout << "TRIANGULO: " << (a * c) / 2 << endl;
   cout << "CIRCULO: " << pi * pow(c, 2) << endl;
   cout << "TRAPEZIO: " << ((a + b) * c) / 2 << endl;
   cout << "QUADRADO: " << pow(b, 2) << endl;
   cout << "RETANGULO: " << a * b << endl;


   return 0;
}

File: python3/beginner/1012.py
a, b, c = input().split(" ")

a = float(a)
b = float(b)
c = float(c)

pi = 3.14159

print("TRIANGULO: {:.3f}".format((a * c) / 2))
print("CIRCULO: {:.3f}".format(pi * (c ** 2)))
print("TRAPEZIO: {:.3f}".format(((a + b) * c) / 2))
print("QUADRADO: {:.3f}".format(b ** 2))
print("RETANGULO: {:.3f}".format(a * b))
